src/main/scala/ch/epfl/bluebrain/nexus/kg/async/RevisionedValue.scala

```diff
@@ -1,17 +1,9 @@
 package ch.epfl.bluebrain.nexus.kg.async
 
-import akka.cluster.ddata.LWWRegister.Clock
-
 /**
   * A value with an attached revision corresponding to the value.
   *
   * @param rev   the value revision
   * @param value the value
   */
 final case class RevisionedValue[A](rev: Long, value: A) extends RegisteredValue[A]
-
-object RevisionedValue {
-
-  private[async] def revisionedValueClock[A]: Clock[RevisionedValue[A]] =
```

Comment:
Why the removal?

Reply:
It wasn't used. When we use it I will add it, but right now is just garbage

Reply:
It is used, but not immediately obvious as creating a register takes a default clock implicitly:
```
/**
   * INTERNAL API
   */
  @InternalApi private[akka] def apply[A](node: UniqueAddress, initialValue: A, clock: Clock[A]): LWWRegister[A] =
    new LWWRegister(node, initialValue, clock(0L, initialValue))

  def apply[A](initialValue: A)(implicit node: Cluster, clock: Clock[A] = defaultClock[A]): LWWRegister[A] =
    apply(node.selfUniqueAddress, initialValue, clock)
```

Also, the only reason this type exists is to be used in a LWWRegister; if the clock is "garbage" the whole type is "garbage".